bpf/lib/nodeport.h

```diff
@@ -1360,11 +1360,73 @@ static __always_inline int dsr_set_opt4(struct __ctx_buff *ctx,
 }
 #endif /* DSR_ENCAP_MODE */
 
-static __always_inline int handle_dsr_v4(struct __ctx_buff *ctx, bool *dsr)
+static __always_inline int handle_dsr_v4(struct __ctx_buff *ctx, bool *dsr, int ct)
```

Comment:
We recently introduced an enumeration for the conntrack states:
```suggestion
static __always_inline int
handle_dsr_v4(struct __ctx_buff *ctx, bool *dsr, enum ct_status ct)
```

Reply:
fixed